code/datums/emergency_calls/emergency_call.dm

```diff
@@ -83,7 +83,19 @@
 	if(SSticker?.mode?.waiting_for_candidates) //It's already been activated
 		return FALSE
 
-	picked_call.mob_max = rand(5, 15)
+	if(name == "Distress Signal") //If we're fighting benos
+		var/xenos = length(GLOB.alive_xeno_list)
+		var/humans = length(GLOB.alive_human_list)
+
+		if(humans / max(1,xenos) > 3) //Humans don't need any further help as they already have advantageous/desirable numbers
```

Comment:
This is already handled elsewhere, having the early return here will fuck up things.